src/test/scala/scala/concurrent/stm/TSetSuite.scala

```diff
@@ -358,7 +358,6 @@ class TSetSuite extends FunSuite {
   test("view builder magic") {
     val s0 = TSet.View(1, 2, 3)
     val s1 = s0 map { "x" + _ }
-    val s2: TSet.View[String] = s1
```

Comment:
This is a no-op line which I removed. Perhaps you wanted to check that the type of `s1` was correct? Then I could add the line again, perhaps with a 'test' call so `s2` is referenced.